examples/scaling-criteo/03-Training-with-HugeCTR.ipynb

```diff
@@ -311,9 +311,12 @@
     "        top_names=[\"loss\"],\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        source=["/raid/data/criteo/test_dask/output/train/_file_list.txt"],
</pre><p>inconsistent w/ use of <code>os.path.join</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>